### ISSUE TITLE
Add annotation sufficiency score to display

### ISSUE THREAD

**Issue by jmcmurry** [TheJacksonLaboratory/PhenopacketLab#151]
We need to give folks realtime feedback about how sufficient their phenotype profile is. There is an api somewhere we can call, but it is somewhat primitive. On the frontend, I recommend instantiating as a star rating.

**Comment by pnrobinson** [comment]
I agree with this but the previous sufficiency meter was not doing exactly what we want and is probably not much better than saying you need to enter X terms. We want to work on an improved algorithm in the Exomiser project this year

**Comment by jmcmurry** [comment]
Awesome, I'd love to see this API updated. Do you think including the current primitive version is worse than having nothing there for feedback?

**Comment by pnrobinson** [comment]
fine to include the current version, just let's make sure we do not depend on its API too much!

**Comment by jmcmurry** [comment]
agreed; if it should become unworkable for any reason, rather than prop up the existing API we could just use term count in the meantime
